src/components/Layout/index.tsx

```diff
@@ -22,7 +23,9 @@ const Layout: FC<ComponentProps> = ({children}) => {
       <div className="Layout__top-nav-wrapper">
         <TopNav className="Layout__TopNav" />
       </div>
-      <div className={clsx({Layout__content: !isHomepage, Layout__home: isHomepage})}>{children}</div>
+      <div className={clsx({Layout__content: !(isHomepage || isProfilePage), Layout__home: isHomepage})}>
```

Comment:
@lapstjup may I ask why is this change needed? after your modifications on this, the profile page no longer respects the 1280px max width and its width is taking the entire page

Reply:
Yeah so I did this to be consistent with Figma design. The light blue background is across the whole width. Does this page need to respect that 1280px max width for any use case ?

Reply:
Oh right, to me it looks slightly weird when on larger screens, also it stays consistent with the leaderboard page (also contains tasks completed) which also has max width of 1280px

Reply:
Let's keep it for now for consistency sake.

Reply:
tested and works fine. thanks for the help

Reply:
No worries. Thanks a ton for confirming. Now go sleep pretty late man 😴 .